site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,99 @@
+    2. If you don’t have time to fix the mistake, copy the file path you’re on and create a GitHub issue about the problem you found. Make sure to include the file path and fill out the issue template completely to maximize clarity.
+    3. If you’re not up for creating a GitHub issue right now, that’s alright too! In the bottom right corner of every docs page is the question “Did you find what you were looking for?” Use this to quickly provide direct feedback about any page you’re viewing.
+    4. If you want to fix a larger problem that affects multiple pages or the structure of the docs, you should first report it as an issue on the appropriate GitHub repo, and follow the steps under section 1, subsection 1 of this guide, “You’ve found a bug”. The developer and user docs repositories contain instructions on how to build and modify the sites locally so you can test larger changes more efficiently.
+    5. The best place to discuss problems with us is in the [Documentation Working Group channel](https://community.mattermost.com/core/channels/dwg-documentation-working-group) where you can ping our technical writers with the hashtag #docsteam.
+3. If you’d like to contribute to our blog, website, or social media, you also have a few options:
+    1. You can get paid to write technical content for developers through the Mattermost [community writing program](https://mattermost.com/blog/blog-announcing-community-writing-program/).
+    2. If you see a problem with any webpages, blog posts, or other content on [Mattermost.com](http://Mattermost.com), you can notify us via [the Content channel](https://community.mattermost.com/core/channels/mattermost-blog).
+    3. Share your contributor or user experience! Mention us when you promote work you do in our community and we’ll amplify the message through Mattermost social media profiles.
+    4. Want to lead a social community? We can provide advice and resources to help you in the [Community channel](https://community.mattermost.com/core/channels/community-team).
+
+## Subsection 3 — You’d like to make something more inclusive or accessible
+
+1. Accessibility is one of the most overlooked yet most important features of modern software development, and we’re eager to improve on the accessibility of the features in our open source project and its documentation.
+2. Problems with the accessibility or the inclusivity of both features in the codebase and pieces of content describing Mattermost are bugs, so we treat them as such in our development process. See subsections 1 and 2 for the description on how you can submit your contribution.
+3. When you contribute a change that incorporates an adjustment based on the principles of accessibility and inclusivity, please use this guide to back you up in the PR, ticket, or post.
```

Comment:
Why would someone need to back up their PR if it's addressing a shortfall in accessibility and/or inclusivity?

Reply:
@neflyte - Given the volume of feedback outstanding, let's revisit this content together prior to merging this PR.